nextcord/permissions.py

```diff
@@ -557,6 +557,14 @@ def moderate_members(self) -> int:
         """
         return 1 << 40
 
+    @flag_value
+    def use_clyde_ai(self) -> int:
+        """:class:`bool`: Returns ``True`` if a user can interact with the Clyde AI bot.
+
+        .. versionadded:: 2.6
```

Comment:
```suggestion
        .. versionadded:: 3.0
```

Reply:
I'll update this when the Discord PR gets merged, just in case we end up having a 3.1 and the docs pr only gets merged then